plover/gui_qt/main.py

```diff
@@ -49,8 +77,9 @@ def __init__(self, config, use_qt_notifications):
         translations_dir = QLibraryInfo.location(QLibraryInfo.TranslationsPath)
         self._translator.load('qtbase_' + get_language(), translations_dir)
         self._app.installTranslator(self._translator)
-        locale.setlocale(locale.LC_ALL, 'en_US')
 
+        default_local = locale.getdefaultlocale()
+        locale.setlocale(locale.LC_ALL, LANGUAGES[default_local[0]])
```

Comment:
No, better just to use:

`local.setlocale(locale.LC_ALL, '')`

If you do it this way, now on Mac and Linux you'll be handing in the wrong thing. Seems like the above is the way to go. If for some reason 'en_CA' doesn't support commas in the numbers, I guess we'll just have to live with that.